Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -132,12 +132,6 @@ protected override void AddNewFileNodeToHierarchy(HierarchyNode parentNode, stri
             {
                 // enable TypeScript on the project automatically...
                 SetProjectProperty(NodeProjectProperty.EnableTypeScript, "true");
-                SetProjectProperty(NodeProjectProperty.TypeScriptSourceMap, "true");
```

Comment:
This properties will no longer serve a purpose because they configure the TypeScript SDK, which we no longer use.

It will be incompatible with older templates or with projects not containing a `tsconfig.json` file. We'll still be able to open the project but build will fail informing to the user the need for a `tsconfig.json` file.

I'm not sure if this clarifies your questions.

Reply:
Comment is outdated since the change is now backwards compatible.